openfisca_uk/calibration/losses/loss_category.py

```diff
@@ -1,8 +1,16 @@
-import tensorflow as tf
-from openfisca_uk import Microsimulation
-from typing import Callable, Iterable, List, Tuple
-from numpy.typing import ArrayLike
+import numpy as np
 from openfisca_core.parameters import ParameterNode, Parameter
+from numpy.typing import ArrayLike
+from itertools import chain
+from functools import reduce
+from typing import Callable, Iterable, List, Tuple, Type
+from openfisca_uk import Microsimulation
+import tensorflow as tf
+import logging
+import os
+
+os.environ["TF_CPP_MIN_LOG_LEVEL"] = "3"
```

Comment:
What does this do?

Reply:
It's a failed attempt to fix https://stackoverflow.com/questions/35911252/disable-tensorflow-debugging-information, nothing functional so will remove.